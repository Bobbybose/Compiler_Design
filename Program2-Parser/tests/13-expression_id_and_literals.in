alpha;
1;
2.2e-3;
'a';
"beta";
