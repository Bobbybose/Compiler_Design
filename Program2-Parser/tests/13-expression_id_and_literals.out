statementList: 1,1
|   
+-- statementList (EMPTY STRING): 1,1
|   
+-- statement: 1,1
|   |   
|   +-- expression: 1,1
|   |   |   
|   |   +-- alpha: 1,1
|   |   |   
|   |   *   
|   |   
|   +-- ;: 1,6
|   |   
|   *   
|   
*   
statementList: 2,1
|   
+-- statementList (EMPTY STRING): 2,1
|   
+-- statement: 2,1
|   |   
|   +-- expression: 2,1
|   |   |   
|   |   +-- 1: 2,1
|   |   |   
|   |   *   
|   |   
|   +-- ;: 2,2
|   |   
|   *   
|   
*   
statementList: 3,6
|   
+-- statementList (EMPTY STRING): 3,6
|   
+-- statement: 3,1
|   |   
|   +-- expression: 3,1
|   |   |   
|   |   +-- 2.2e-3: 3,1
|   |   |   
|   |   *   
|   |   
|   +-- ;: 3,7
|   |   
|   *   
|   
*   
statementList: 4,3
|   
+-- statementList (EMPTY STRING): 4,3
|   
+-- statement: 4,1
|   |   
|   +-- expression: 4,1
|   |   |   
|   |   +-- 'a': 4,1
|   |   |   
|   |   *   
|   |   
|   +-- ;: 4,4
|   |   
|   *   
|   
*   
statementList: 5,6
|   
+-- statementList (EMPTY STRING): 5,6
|   
+-- statement: 5,1
|   |   
|   +-- expression: 5,1
|   |   |   
|   |   +-- "beta": 5,1
|   |   |   
|   |   *   
|   |   
|   +-- ;: 5,7
|   |   
|   *   
|   
*   
